inference-engine/thirdparty/clDNN/src/program.cpp

```diff
@@ -1374,4 +1374,9 @@ void program::set_layout_optimizer_attributes(layout_optimizer& lo) {
 
     if (should_use_bs_fs_yx_bsv16_fsv16)
         lo.set_optimization_attribute(layout_optimizer::optimization_attributes_type::bs_fs_yx_bsv16_fsv16_network, 1);
+
+#ifdef ENABLE_ONEDNN_FOR_GPU
```

Comment:
just out of curiosity: why did you change the spec of `set_layout_optimizer_attributes`? I think it is possible to call `get_engine` within the function.

Reply:
You are absolutely right! My fault